blueocean-web/src/main/js/try.js

```diff
@@ -1,6 +1,19 @@
 var $ = require('jquery-detached').getJQuery();
 var jsModules = require('@jenkins-cd/js-modules');
 
+function getAppUrl() {
+    var rootUrl = jsModules.getRootURL();
+    
+    // If there's no prefix, rooturl is blank, resulting in js-modules
+    // returning the static url. We don't want to use that when
+    // switching to blue ocean, so return '' instead.
+    if (rootUrl.indexOf('/static/') === 0) {
```

Comment:
... I mean, why would js-modules ever return the static URL as the root URL? this isn't normal behavior. handling it here is the wrong place, fix js-modules to not do this... if you need that static resource url, create a method for it if there isn't one.